### ISSUE TITLE
 2.2.2 Pause, Stop, Hide - Definition of term "automatic" needed

### ISSUE THREAD

**Issue by dan-tripp-siteimprove** [w3c/wcag#2863]
2.2.2 applies to information that "starts automatically".  It's unclear to me whether the following cases count a "automatic": 

1. When the user moves focus to a certain button, a video starts to play. 
2. When the user hovers their mouse over a certain region of the screen, a video starts to play. 

If WCAG included a definition of "automatic", maybe it should include the idea of the intention of the user, or what the web page advertised to the user.  eg. if a typical user would not expect a video their change of keyboard focus to cause a video to start playing, then that was automatic.

**Comment by patrickhlauke** [comment]
Agree. Up to now, the general understanding/implication of "automatic" has been "without user intervention/interaction"

Arguably, a user hovering over something, and moving focus somewhere, would traditionally count as a user interaction, so in that light, it would not fall under the SC... but this would be a real-world problem for users.

Unfortunately, it's likely not easy to unambiguously define the "[a] typical user would not expect" part though, and I suspect that's going to be a sticking point for a binary pass/fail SC. but it would be interesting to get other WG members' opinions here.

Note that the same would also apply to the use of "automatic" for 1.4.2 Audio Control (but arguably, NOT for the use of "automatic" in 3.2.2 On Input, 3.3.1 Error Identification, 3.3.3 Error Suggestion)

**Comment by alastc** [comment]
> the general understanding/implication of "automatic" has been "without user intervention/interaction"

I agree with that, and that was why we added [Animation from interactions](https://www.w3.org/TR/WCAG22/#animation-from-interactions), because that wasn't automatic.

If we do add a definition of automatic (either normatively, or more likely by explaining it in the understanding docs), it will need to be fairly narrow. (From experience, this is what happens when there are multiple interpretations: The narrow one is taken so as not to cause issues for people where suddenly things seem to fail where they didn't before.)

In this case it doesn't seem like (many) people would have been interpreting automatic as involving user-interaction, so it shouldn't be a problem.

**Comment by dan-tripp-siteimprove** [comment]
I'm glad that you pointed out 2.3.3 Animation from Interactions.  It had escaped my attention.

[Understanding 2.3.3](https://www.w3.org/WAI/WCAG22/Understanding/animation-from-interactions.html) mentions 2.2.2: "In contrast, [2.2.2 Pause, Stop, Hide](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide.html) applies when the web page initiates animation.".  I wonder if [Understanding 2.2.2](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide.html) would benefit from mentioning 2.3.3?  (Currently it mentions 2.3 but not 2.3.3.)

As for what 2.3.3 covers, it seems not to cover video, because it covers only "[motion animation](https://www.w3.org/WAI/WCAG22/Understanding/animation-from-interactions.html#dfn-motion-animation)" which is "addition of steps between conditions to create the illusion of movement or to give a sense of a smooth transition".  This seems odd to me because a full-blown video is, I'm guessing, at least as harmful to the users in question as a 'motion animation'.

I don't know how to fix everything discussed here, but what would you both think of a pull request that included the items below? 
* Adding to 2.2.2 a narrow definition of "automatic" 
* Adding to 2.3.3 a link to 2.2.2.

**Comment by alastc** [comment]
Hi @dan-tripp-siteimprove,

That seems like a good next step, thank you. I'd suggestion:
- Forking the repo (you suggested a PR so I'm assuming you're familiar with general git/github processes). Or if you have access, you can create a branch here, e.g. `Issue2863_automatic-animation`
- Editing the understanding > 20 > pause-stop-hide.html file, finding a suitable place in the Intent section to explain what automatic means in this context.
- Adding the cross-reference to 2.3.3.
- Open the PR, and include "Closes 2963" in the description.

**Comment by dan-tripp-siteimprove** [comment]
Okay, I'll try to do that soon.

**Comment by dan-tripp-siteimprove** [comment]
Done.

**Comment by dan-tripp-siteimprove** [comment]
In [my PR for this](https://github.com/w3c/wcag/pull/2906), in the interest of avoiding the situation "where suddenly things seem to fail where they didn't before": I made my definition of "automatic" so narrow that it's not very useful, now that I think about it.  Do either of you think that we could broaden the definition of "automatic" to include pointer hover and/or scrolling the page?

**Comment by patrickhlauke** [comment]
I've made a counter-proposal to address this here https://github.com/w3c/wcag/pull/4012
